engine/src/main/java/org/terasology/rendering/dag/nodes/InitialPostProcessingNode.java

```diff
@@ -74,23 +71,13 @@
     @Range(min = 0.0f, max = 1.0f)
     private float bloomFactor = 0.5f;
 
-    private SubmersibleCamera activeCamera;
-    @SuppressWarnings("FieldCanBeLocal")
-    private Vector3f sunDirection;
-    @SuppressWarnings("FieldCanBeLocal")
-    private Vector3f cameraDir;
-    @SuppressWarnings("FieldCanBeLocal")
-    private Vector3f cameraPosition;
     @SuppressWarnings("FieldCanBeLocal")
     private Vector3f tint;
 
     public InitialPostProcessingNode(Context context) {
-        backdropProvider = context.get(BackdropProvider.class);
         renderingConfig = context.get(Config.class).getRendering();
         worldProvider = context.get(WorldProvider.class);
-        worldRenderer = context.get(WorldRenderer.class);
-
-        activeCamera = worldRenderer.getActiveCamera();
+        activeCamera = context.get(WorldRenderer.class).getActiveCamera();
 
         DisplayResolutionDependentFBOs displayResolutionDependentFBOs = context.get(DisplayResolutionDependentFBOs.class);
         // TODO: see if we could write this straight into a GBUFFER - notice this FBO is used in ShaderParametersHdr
```

Comment:
So what do you recommend? Just removing the comment and continue writing to INITIAL_POST_FBO, or remove comment and remove this buffer and write directly to GBUFFER?